docs/smart-contracts/quickstart.md

```diff
@@ -61,6 +61,20 @@ curl --proto '=https' --tlsv1.2 -LsSf https://github.com/near/cargo-near/release
 
 </TabItem>
 
+<TabItem value="py" label="🐍 Python">
+
+```bash
+# Install Python (if not already installed)
+# Use your system's package manager or download from https://www.python.org/downloads/
+
+# Install uv for Python package management
+curl -LsSf https://astral.sh/uv/install.sh | sh
+
+# Install NEAR CLI-RS to deploy and interact with the contract
+curl --proto '=https' --tlsv1.2 -LsSf https://github.com/near/near-cli-rs/releases/latest/download/near-cli-rs-installer.sh | sh
+```
+
```

Comment:
It needs to be added that `emscripten` has to be installed so the Python contract can be compiled.

Currently it's not mentioned anywhere, and it will bring errors later when following the tutorial.